pages/common/git-subtree.md

```diff
@@ -5,19 +5,19 @@
 
 - Add a git repository as a subtree:
 
-`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{master}}`
+`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{branch_name}}`
```

Comment:
Is the final parameter mandatory? Maybe we should omit it in most examples if it defaults to the primary branch, which I'd suppose is what most people would want.

Reply:
I would say that is outside the scope of this PR.

Reply:
It doesn't hurt to review it now I don't think, @qvissak.

Anyway, it looks like only the repository URL is optional according to the man page @waldyrious, as I see the following in the usage summary at the top:

```
git subtree add   -P <prefix> <commit>
git subtree add   -P <prefix> <repository> <ref>
```

You'd have thought it would be the commit ref that's optional, but it's the repository url instead :woman_shrugging:

Interestingly we mention here `branch_name`, but it's actually more like a `commit_ish` according to the man page. That said, I havevn't actually used this command personally, so I wouldn't know.

Reply:
Marking as resolved, but feel free to unresolve if this isn't the case.